### ISSUE TITLE
Not able to open chests, hit shrines etc in multiplayer

### ISSUE THREAD

**Issue by kgDraco** [FelixVolo/ActuallyFaster#13]
Not able to open chests, use shrines of chance or blood, etc. Disabling this mod fixes the issue. Issue in single player and multiplayer. 
VIdeo:
https://github.com/user-attachments/assets/34f00d5d-a7b1-471f-ad6b-3abd7f1ff74d

Mod list:
0192033c-d2af-d339-55a0-2e8a533f6429

**Comment by FelixVolo** [comment]
I imported your profile and indeed it didn't work.  BUT even BepInEx wasn't working correctly. Idk what you did to your config and at this point i don't want to. I created a profile with exactly the same mods and "somehow" my mod works. 

0192041f-21c3-ecf2-e90b-1c34fd02cef8

**Comment by kgDraco** [comment]
Thanks for taking the time to validate, and sorry I didn't mention it before - thanks for making this mod!
After testing with the code you provided (and your mod in isolation), the issue occurs if all of these flags are set to false:

-Chance Shrine brrrrrt=false
-Cauldron brrrrrt=false
-Cleansing Pool brrrrrt=false

Only occurs if BepInExPack is enabled; tested with & without other mods. Does not seem to matter if the other flags are true or false. 
Turns out this bug is an edge case that I hit since I used this mod for only printers and scrappers, so I had set all else to false.
